storage/storagedriver/select-storage-driver.md

```diff
@@ -55,7 +55,7 @@ Docker supports the following storage drivers:
    driver is poor, and is not generally recommended for production use.
 
 Docker's source code defines the selection order. You can see the order at
-[the source code for Docker Engine - Community {{ site.docker_ce_version }}](https://github.com/docker/docker-ce/blob/{{ site.docker_ce_version }}/components/engine/daemon/graphdriver/driver_linux.go#L50)
```

Comment:
@thaJeztah Should this go to https://github.com/moby/moby/blob/20.10/daemon/graphdriver/driver_linux.go#L50 now?